src/Repository/VVTRepository.php

```diff
@@ -15,55 +16,67 @@
  */
 class VVTRepository extends ServiceEntityRepository
 {
-    public function __construct(ManagerRegistry $registry)
+    public function __construct(
+        ManagerRegistry $registry,
+        private readonly TeamRepository $teamRepository,
+    )
     {
         parent::__construct($registry, VVT::class);
     }
 
-    public function findActiveByTeam($value)
+    public function findActiveByTeam(Team $team)
     {
-        return $this->createQueryBuilder('a')
-            ->andWhere('a.team = :val')
-            ->andWhere('a.activ = 1')
-            ->setParameter('val', $value)
-            ->orderBy('a.CreatedAt', 'DESC')
-            ->getQuery()
-            ->getResult()
-            ;
+        $queryBuilder = $this->getBaseQueryBuilder($team);
+        $this->excludeIgnored($team, $queryBuilder);
+        return $queryBuilder->getQuery()->getResult();
     }
```

Comment:
Even when datenweitergabe is inherited from a parent team and the inheriting team has disabled the use of the parents vvt in it's child team, the vvt should be listed in the readonly form of the inherited datenweitergabe object.

Steps to reproduce:
1. Create two teams: parent -> child
2. Create vvt in parent team and active inheritance checkbox
3. Create Datenweitergabe in parent team and connect it to vvt
4. Now go child team, select the inherited vvt and click on "Disable for this team"
5. Go to the inherited datenweitergabe, open the form and check that the value of the vvt form field is missing

![image](https://github.com/H2-invent/open-datenschutzcenter/assets/47010475/49e06bc6-b325-4baa-9212-a834682c6bcf)